client/src/components/VideoBackground/VideoBackground.css

```diff
@@ -0,0 +1,11 @@
+video.fullscreen { 
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    min-width: 100%;
+    min-height: 100%;
+    width: auto;
+    height: auto;
```

Comment:
I think you can just use min-width: 100% and min-height: 100% without the width auto or height auto. I may be wrong, I don't know how your page is styled and if you have a good reason for doing it like this, then by all means keep it. [](https://stackoverflow.com/questions/41987305/what-does-min-width-100-width-auto-actually-do) this is what I read. If you need to override some other css, then keep it the way it is.